public/components/metrics/sidebar/index_picker.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiComboBox, EuiTitle } from '@elastic/eui';
+import React, { useState } from 'react';
+
+export const IndexPicker = (props) => {
+  const { otelIndices, setSelectedOTIndex } = props;
```

Comment:
convert to redux dispatch/useselector

Reply:
these are used just in couple of files so state change made sense